GitCommands/RevisionReader.cs

```diff
@@ -19,18 +19,23 @@ namespace GitCommands
     [Flags]
     public enum RefFilterOptions
     {
-        None                    = 0x000,
-        Branches                = 0x001,    // --branches
-        Remotes                 = 0x002,    // --remotes
-        Tags                    = 0x004,    // --tags
-        Stashes                 = 0x008,    //
-        All                     = 0x00F,    // --all
-        Boundary                = 0x010,    // --boundary
-        ShowGitNotes            = 0x020,    // --not --glob=notes --not
-        NoMerges                = 0x040,    // --no-merges
-        FirstParent             = 0x080,    // --first-parent
-        SimplifyByDecoration    = 0x100,    // --simplify-by-decoration
-        Reflogs                 = 0x200,    // --reflog
+        None = 0x000,
+        Branches = 0x001, // --branches=<filter>
+
+        // Unused
+        // Remotes = 0x002, // --remotes
+        // Tags = 0x004, // --tags
+        All = 0x007, // --all (default is HEAD)
+
+        // Exclude some refs from --all
+        NoStash = 0x008, // --exclude=refs/stash
+        NoGitNotes = 0x010, // --not --glob=notes --not
+
+        NoMerges = 0x020, // --no-merges
+        FirstParent = 0x040, // --first-parent
+        SimplifyByDecoration = 0x080, // --simplify-by-decoration
+        Boundary = 0x100, // --boundary
+        Reflogs = 0x200, // --reflog
```

Comment:
Please keep indentation

Reply:
Msvs keeps changing at every edit so it is quite annoying to retain spaces...

Reply:
Can we agree on this?
This is a (minor) regression for 4.0, but also one dependency for stash-in-grid that I will suggest to be included in 4.0 too...

Reply:
Rebased without changes